src/star--0.css

```diff
@@ -0,0 +1,10 @@
+.stars--0 .stars__star:nth-child(-n + 5) {
+  background-image: url(/src/images/star.svg);
+  background-size: 13px;
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
You can just put this into `stars_star` selector. And then just overwrite the `background-image` property with active star image in selector with `nth-child` pseudo class.